host_os.py

```diff
@@ -34,6 +35,7 @@
     'build-package': build_package,
     'release-notes': create_release_notes,
     'upgrade-versions': upgrade_versions,
+    'update-versions-readme': update_versions_readme,
```

Comment:
This name is inconsistent with "upgrade-versions" above. I prefer to use "update" for everything, but that would require changing the infrastructure repo too. We can leave this change for later if you feel it's too much trouble for now.